src/app/@theme/components/job-actions-menu/job-actions-menu.component.ts

```diff
@@ -0,0 +1,109 @@
+import { Component, OnInit, OnDestroy } from '@angular/core';
+import { ActivatedRoute, Router } from '@angular/router';
+import { Subscription } from 'rxjs';
+import { JobService } from '../../../@core/services/job.service';
+import { ToastService } from 'src/app/@core/services/toast.service';
+import { Job } from '../../../@core/models/models';
+
+@Component({
+  selector: 'intelowl-job-actions-menu',
+  templateUrl: './job-actions-menu.component.html',
+  styleUrls: ['./job-actions-menu.component.scss'],
+})
+export class JobActionsMenuComponent implements OnInit {
+  // Job ID whose result is being displayed
+  public jobId: number;
+
+  // interval var
+  private pollInterval: any;
```

Comment:
all the code related to polling is not needed in this component.

Line 18, 37, 105

Reply:
what about the subscription code in `ngOnInit(): void` ? Should only the polling part be removed or the entire thing? And if we aren't subscribing, what will come in `ngOninit()` instead?

Reply:
the polling part will stay as is in JobResult and is not required in actions component. ngOnInit and ngOnDestroy can be removed from the actions component.